src/discord/events/message/index.ts

```diff
@@ -27,48 +28,55 @@ export type SubCommands = Record<string, HandlerFn>;
 export const commands: Command[] = [
   {
     handlerId: 'help',
-    trigger: '!help',
-    description: stringDictionary.helpDescript,
```

Comment:
Please revert all the changes that replaced the description with hard coded text instead of using he string dictionary